tests/moment-timezone/parse.js

```diff
@@ -251,6 +251,23 @@ exports.parse = {
 		t.done();
 	},
 
+    "check that moment(date).tz(timezone) and moment.tz(date, timezone) behave the same way" : function (t) {
+        var sourceTimeZone = 'America/Los_Angeles';
```

Comment:
Lot lot lot of people think that 
```moment(date).tz(timezone) and moment.tz(date, timezone)``` 
should work in a same way.
We should definitely update the docs